xsdValid.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python
+
+import xmlschema
+
+format1 = xmlschema.XMLSchema('xsd/package_format1.xsd')
+format2 = xmlschema.XMLSchema('xsd/package_format2.xsd')
+format3 = xmlschema.XMLSchema('xsd/package_format3.xsd')
+
+print("Format1: ", format1.validity)
+print("Format2: ", format2.validity)
+print("Format3: ", format3.validity)
```

Comment:
Just printing the validity value doesn't seem to be helpful in the context of a CI build which should ensure that is actually is valid.

Reply:
The python script uses `xmlschema` to load and validate each file. If the file is invalid, an error is generated and the build fails.

Reply:
The API doc of `validity` says:

> Property that returns the XSD validator's validity. It can be ‘valid’, ‘invalid’ or ‘notKnown’.

Reply:
Correct, but before printing the content of the `validity` flag, my script loads them using `xmlschema.XMLSchema` which is what does the actual validation and will fail with an error if it fails to pass. The print of the `validity` flag is just a double-check.

Reply:
If you think it would be more "proper," I can add some try blocks around each load and then print to stderr if one fails, rather than letting the script crash on a failed validation.

Reply:
Then the print statement is pretty misleading since from reading the code you would assume that it prints the validity state.

Please change the print statements to just say:

> 'xsd/package_format1.xsd' file is valid

Also please move each print statement directly after each `XMLSchema` constructor. Then if format 2 fails at least the message that format 1 passed has been printed already.